simulator/opendc-experiments/opendc-experiments-allocateam/src/main/kotlin/org/opendc/experiments/allocateam/telemetry/writers/ParquetRunCompletedEventWriter.kt

```diff
@@ -28,7 +28,8 @@ public class ParquetRunCompletedEventWriter(path: File) :
             record.put("repetitions", scenario.repetitions)
             record.put("topology", scenario.topology.name)
             record.put("workload_name", scenario.workload.name)
-            record.put("allocation_policy", scenario.allocationPolicy)
+            record.put("resource_selection_policy", scenario.resourceSelectionPolicy)
```

Comment:
This is the format that is used to generate the plots, if you change it, you also have to change the plotting code 🙃  So you can either just have an `allocation_policy` that or you add a check to differentiate how your code is handled from the other policies.